apps/api/test/functional/auth0.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { UserSetting } from "@akashnetwork/database/dbSchemas/user/userSetting";
+import { faker } from "@faker-js/faker";
+import type { GetUsers200ResponseOneOfInner } from "auth0";
+import { container } from "tsyringe";
+
+import { Auth0Service } from "@src/auth/services/auth0/auth0.service";
+import { ExecutionContextService } from "@src/core/services/execution-context/execution-context.service";
+import { app } from "@src/rest-app";
+import { UserRepository } from "@src/user/repositories/user/user.repository";
+
+import { UserSeeder } from "@test/seeders/user.seeder";
+
+jest.setTimeout(30000);
+
+describe("Auth0 Email Verification", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
one of the point of using `setup` function is that jest hooks aren't needed as every test case gets it's own clean state